internal/controllers/clusterextension_controller.go

```diff
@@ -237,6 +237,7 @@ func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1alp
 	setResolvedStatusConditionSuccess(ext, fmt.Sprintf("resolved to %q", resolvedBundle.Image))
 
 	bundleSource := &rukpaksource.BundleSource{
+		Name: ext.GetName(),
```

Comment:
Seems like we must have a bug if we're not setting this. I think it means that:
1. All cluster extensions unpacking is happening in the same directory
2. Because of (1), only the most recently reconciled bundle will have its FS extracted at once.
3. Because of (2), when a ClusterExtension reconciles it has to re-pull and re-unpack its image if another ClusterExtension has reconciled in the meantime.
4. (Not a problem today) If we turned on concurrent reconciles, then things would likely break even harder because the unpacker would be racing to delete the what it thinks are unused unpack paths while other reconciles are writing to them for other ClusterExtensions.

Reply:
This this in a separate PR?

Reply:
https://github.com/operator-framework/operator-controller/pull/1243